.openshift-ci/jobs/apollo-src/Dockerfile

```diff
@@ -0,0 +1,18 @@
+FROM replaced-by-osci:bin AS pr-data
+
+RUN .openshift-ci/jobs/apollo-src/dump-pr-data.sh
+
+FROM quay.io/stackrox-io/apollo-ci:collector-0.3.44-1-gb00ffc52af
```

Comment:
As above - should this use `pipeline:apollo-ci`?

Reply:
See https://github.com/stackrox/collector/pull/826#discussion_r983375354